deploy/crds/ssp.kubevirt.io_v1_ssp_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: ssp.kubevirt.io/v1
+kind: SSP
+metadata:
+  name: example-ssp
+spec:
+  # Add fields here
+  size: 3
```

Comment:
Please remove spec fields and use `spec: {}` instead

Reply:
Done